Sources/StreamChat/Repositories/AuthenticationRepository.swift

```diff
@@ -75,25 +76,12 @@ class AuthenticationRepository {
         get { tokenQueue.sync { _tokenProvider } }
         set { tokenQueue.async(flags: .barrier) { self._tokenProvider = newValue }}
     }
-
-    private var tokenRequestCompletions: [(Error?) -> Void] {
-        get { tokenQueue.sync { _tokenRequestCompletions } }
-        set { tokenQueue.async(flags: .barrier) { self._tokenRequestCompletions = newValue }}
-    }
-
-    /// An array of requests waiting for the token
-    private(set) var tokenWaiters: [String: (Result<Token, Error>) -> Void] {
-        get { tokenQueue.sync { _tokenWaiters } }
-        set { tokenQueue.async(flags: .barrier) { self._tokenWaiters = newValue }}
-    }
-
+    
     weak var delegate: AuthenticationRepositoryDelegate?
 
     private let apiClient: APIClient
     private let databaseContainer: DatabaseContainer
     private let connectionRepository: ConnectionRepository
-    /// Retry timing strategy for refreshing an expired token
-    private var tokenExpirationRetryStrategy: RetryStrategy
```

Comment:
Uses `_` now to signify that it must be guarded with the `tokenQueue`. RetryStrategy has mutating functions so we must synchronise it.